src/main.cc

```diff
@@ -1006,9 +1028,10 @@ mainReconfigureFinish(void *)
 #endif
     }
 
-    serverConnectionsOpen();
+    if (IamWorkerProcess())
+        startWorkerServices();
 
-    neighbors_init();
+    serverConnectionsOpen();
```

Comment:
Please encapsulate this duplicated non-trivial logic into a new startServices() function. If you can move more calls there (e.g., wccp*Init()), please do that but only if they can be moved "as is" -- we should cleanup as much as possible without becoming responsible for out-of-scope service order/initiation changes.

Reply:
Done.